examples/analytics/emr-on-eks-fargate/main.tf

```diff
@@ -53,6 +53,18 @@ module "eks" {
     coredns = {
       configuration_values = jsonencode({
         computeType = "Fargate"
+        # Ensure that the we fully utilize the minimum amount of resources that are supplied by
+        # Fargate https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html
```

Comment:
I think is good to include this other link https://docs.aws.amazon.com/eks/latest/userguide/fargate-pod-configuration.html
Don't know if you want to add this link or replace the current one with this one.

This one shows that 256MB are being reserved for kubelet, kube-proxy, containerd
>Fargate adds 256 MB to each pod's memory reservation for the required Kubernetes components (kubelet, kube-proxy, and containerd).
Fargate rounds up to the following compute configuration that most closely matches the sum of vCPU and memory requests in order to ensure pods always have the resources that they need to run.

And also provides an example 
>The additional memory reserved for the Kubernetes components can cause a Fargate task with more vCPUs than requested to be provisioned. For example, a request for 1 vCPU and 8 GB memory will have 256 MB added to its memory request, and will provision a Fargate task with 2 vCPUs and 9 GB memory, since no task with 1 vCPU and 9 GB memory is available.